app/lib/components/term.js

```diff
@@ -149,6 +160,10 @@ export default class Term extends Component {
     if (this.props.customCSS !== nextProps.customCSS) {
       this.term.prefs_.set('user-css', this.getStylesheet(nextProps.customCSS));
     }
+
+    if (this.devicePixelRatio !== window.devicePixelRatio) {
+      this.term.prefs_.set('font-smoothing', this.getFontSmoothing());
```

Comment:
Right, I was thinking about that, but would that transition really guarantee that new props will be received? Or should we hook into some sort of event (if any)? Maybe `resize` triggers? Can you confirm? And if so, we should document that we're relying on that effect.